### ISSUE TITLE
Please implement AutoEject

### ISSUE THREAD

**Issue by Zerogoki00** [MrKinau/FishingBot#28]
It would be nice if this bot could throw away junk items from user-defined list (for example bowls, leather boots etc...) and keep only needed items (enchanted books, bows, etc)

**Comment by MrKinau** [comment]
thanks for your issue.
This could be a useful feature. Atm it should work, that you place hoppers (as in a default afk fishing farm), Maybe it could be extended to drop defined items. A problem with that is the Support of all versions (For 1.8 I used the Bukkit Material-list, because items had numeric Ids; For other versions I extracted the Item IDs of the mcserver (Yes Items still have numeric Ids, but they change in almost each version)). How could a list be configured. For later versions the minecraft:xyz names could be used and for older versions the item ids. Well this sounds poissble. Will implement it if I finally find time again :)
